api.planx.uk/modules/file/service/utils.ts

```diff
@@ -23,7 +23,7 @@ function useMinio() {
     // Points to Minio
     return {
       endpoint: `http://minio:${process.env.MINIO_PORT}`,
-      s3ForcePathStyle: true,
+      forcePathStyle: true,
```

Comment:
this took me a good while to hunt down! a pesky little breaking change resulting from our move to v3 `@aws-sdk/client-s3` in #3989 that was causing 500 errors with the following msg:

```
ServerError: Failed to upload public file: getaddrinfo ENOTFOUND pizza-user-uploads.minio
```

Reply:
Thanks for shaking some of these out - went unnoticed for a good while!